### PR TITLE
Adding instrumentation for traces regarding Image and texture loading and preparation

### PR DESCRIPTION
**Pull request by samcake**
THis pR simply adds instrumentation to get a better understanding of the workload in the image loading and the baking of textures.

Also changed the default Material textures to be usefull when Material Textures are off. this has no impact on the regular behavior.

## Test Plan
Smoke test, nothing should  have changed
Running in test mode with a test script (like this one https://raw.githubusercontent.com/highfidelity/hifi_tests/master/benchmarks/50-avatars.js) then you should see new fields in the trace for the image loading:
image

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 446aaf3bce1f2842af75de4d8dc5940726cd8956 available at: https://gist.github.com/b9af4e6cc386aaf83f0599feab2d3be9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9592-446aaf3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9592-446aaf3.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 10
QA Pass

General smoke test = pass
New default textures (seen with debugDeferredLighting.js) works as expected.
New trace fields present.

File: libraries/model/src/model/TextureMap.cpp [hunk 1]
```diff
@@ -72,7 +73,8 @@ QImage processSourceImage(const QImage& srcImage, bool cubemap) {
     }
 
     if (targetSize != srcImageSize) {
-        qDebug(modelLog) << "Resizing texture from " << srcImageSize.x << "x" << srcImageSize.y << " to " << targetSize.x << "x" << targetSize.y;
+         PROFILE_RANGE(resource_parse, "processSourceImage Rectify");
+         qDebug(modelLog) << "Resizing texture from " << srcImageSize.x << "x" << srcImageSize.y << " to " << targetSize.x << "x" << targetSize.y;
```
Comment (Atlante45) [inline libraries/model/src/model/TextureMap.cpp]: Shouldn't that be a `qCDebug`, I'm not sure why this compiled.

**Comment by samcake** [comment]
I m going to go ahead and merge it since i fixed the only problem seen by clement so we have the intrumentation in master for friday test

**Comment by hifi-gustavo** [comment]
Build console output for commit: c4ec0cc069838643cd444a2ef92fca2cb109952d available at: https://gist.github.com/b9af4e6cc386aaf83f0599feab2d3be9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9592-c4ec0cc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9592-c4ec0cc.dmg

**Pull request merged**